cypress/e2e/createEditTenure.cy.js

```diff
@@ -345,59 +267,44 @@ describe('create and edit tenure', { tags: ['@tenure', '@authentication', '@comm
             cy.contains("Next").click()
 
             modal.modalBody().should("be.visible");
             modal.modalBody().contains("Are you sure you want to change the status of the tenure to active?");
             modal.cancelButton().click({ force: true });
             createTenurePage.tenureTypeSelection().should('be.visible')
-            createTenurePage.tenureStartDateDayContainer().should('be.visible')
-            createTenurePage.tenureStartDateMonthContainer().should('be.visible')
-            createTenurePage.tenureStartDateYearContainer().should('be.visible')
+            createTenurePage.tenureStartDateInput().should('be.visible')
 
             cy.contains("Next").click()
             modal.modalBody().should("be.visible");
             modal.modalBody().contains("Are you sure you want to change the status of the tenure to active?");
             modal.confirmationButton().click();
-            createTenurePage.doneButton().click()
+            createTenurePage.doneButton().click();
             createTenurePage.confirmTenureUpdatedText().should('contain', 'Tenure updated');    
         })
     })
 
-    tenureTypes.forEach((tenureType) => {
-        it('should allow edit to end date for all tenure types', ()=> {
-            const tenureId = "94690f7d-019e-d00c-21aa-d7a5791b1294"
-            cy.getAssetFixture().then(() =>{
-                createTenurePage.editTenure(tenureId);
-
-                createTenurePage.tenureTypeSelection().should('be.visible')
-                createTenurePage.tenureStartDateDayContainer().should('be.visible')
-                createTenurePage.tenureStartDateMonthContainer().should('be.visible')
-                createTenurePage.tenureStartDateYearContainer().should('be.visible')
-                cy.getTenureFixture(({ id: tenureId }) => {
-                    cy.url().should('include', `tenure/${tenureId}/edit`)
-                })
-                createTenurePage.tenureTypeSelection().select(tenureType)
-                
-                createTenurePage.tenureEndDateDayContainer().should('be.enabled')
-                createTenurePage.tenureEndDateMonthContainer().should('be.enabled')
-                createTenurePage.tenureEndDateYearContainer().should('be.enabled')
-            })
-        })
-    }) 
+    it('should allow edit to end date for all tenure types', ()=> {
+        const tenureId = "94690f7d-019e-d00c-21aa-d7a5791b1294"
+        cy.getAssetFixture().then(() =>{
+            createTenurePage.editTenure(tenureId);
 
+            createTenurePage.tenureTypeSelection().should('be.visible')
+            createTenurePage.tenureStartDateInput().should('be.visible')
+            cy.getTenureFixture(({ id: tenureId }) => {
+                cy.url().should('include', `tenure/${tenureId}/edit`)
+            })
 
-    
+            tenureTypes.forEach(tenureType => {
+                createTenurePage.tenureTypeSelection().select(tenureType);

```

Comment:
Do we really need to do all of them? Maybe select a random couple to do? :fearful: